### PR TITLE
small changes to code structures

### PR DESCRIPTION
**Pull request by HeyItsGwen**
Final pull of all features from userCreation into the dev branch after confirmed working. Adds ability to create users, posts associated with users, and comments associated with posts and users.

### CODE REVIEW

File: app/models/comment.rb [hunk 1]
```diff
@@ -1,5 +1,7 @@
 class Comment < ApplicationRecord
+
+  validates :body, length: {maximum: 280, minimum: 1}
   validates :user_id, presence: true
   validates :post_id, presence: true
```
Comment (maelfosso) [inline app/models/comment.rb]: Use `:user` and `:post` instead

File: app/models/post.rb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 class Post < ApplicationRecord
+
   has_many :comments
   validates :title, length: {maximum: 100, minimum: 5}
   validates :body, length: {maximum: 1000, minimum: 15}
   validates :user_id, presence: true
```
Comment (maelfosso) [inline app/models/post.rb]: use `:user` instead

**Comment by HeyItsGwen** [comment]
Changes made, and resubmitting for code review

**Comment by HeyItsGwen** [comment]
Changes made after code review. Added linting files and a descriptive readme, as well as edited the main PR comment to reflect the changes being made.

**Pull request merged**